README.md

```diff
@@ -1 +1,115 @@
 # example-golang
+
+An example repository to demonstrate Pants's experimental Golang support.
+
+See [] for some unique benefits Pants brings to Golang repositories, and see
```

Comment:
TODO

Reply:
Yep, waiting for the blog to have a stable link. Chicken and egg

Reply:
Mm. Maybe just the http://blog.pantsbuild.org in the meantime then?

Reply:
This repo isn't publicized yet, and planning on doing the blog on Monday. Will fix then a couple minutes before we publicize the blog